build_runner_core/lib/src/changes/build_script_updates.dart

```diff
@@ -51,7 +51,7 @@ class _MirrorBuildScriptUpdates implements BuildScriptUpdates {
     try {
       allSources = _urisForThisScript
           .map((id) => idForUri(id, packageGraph))
-          .whereNotNull()
+          .nonNulls
```

Comment:
This was IN the SDK before 3.5.0-...

Why bump the constraint?

Reply:
Analyzer

Reply:
(this bump should have actually landed in https://github.com/dart-lang/build/pull/3719)

Reply:
The SDK min bump is required by analyzer?